samples/subsys/usb/hid-cdc/README.rst

```diff
@@ -0,0 +1,90 @@
+.. _usb_hid-cdc:
+
+USB HID CDC ACM Application
+################################
+
+Overview
+********
+
+This sample app demonstrates use of multiple USB classes with multiple
+instances. It combines two HID instances and two CDC ACM instances.
+This sample can be found under :file:`samples/subsys/usb/hid-cdc` in the
+Zephyr project tree.
+
+Requirements
+************
+
+This project requires an USB device driver.
```

Comment:
I think we probably need to mention that the sample requires many endpoints supported.

Reply:
Added note.